src/send/SendConfirmationLegacy.tsx

```diff
@@ -381,10 +381,7 @@ function SendConfirmationLegacy(props: Props) {
     }
 
     return (
-      <SafeAreaView
-        style={styles.container}
-        edges={props.route.name === Screens.SendConfirmationLegacyModal ? ['bottom'] : undefined}
```

Comment:
same here, why remove?

Reply:
Was needed for the `react-native-safe-area-context` upgrade but is no longer included with 5d947ba8bb9c3a3f0740a60a59c04fa33e52a533.